app/api/utils.py

```diff
@@ -455,6 +455,37 @@ def render(self, data, accepted_media_type=None, renderer_context=None):
                              allow_nan=not self.strict) + '\n'
 
 
+class FastTextPainter(object):
+
+    @staticmethod
+    def paint_labels(documents, labels):
+        serializer = DocumentSerializer(documents, many=True)
+        serializer_labels = LabelSerializer(labels, many=True)
+        data = []
+        for d in serializer.data:
+            labels = []
+            for a in d['annotations']:
+                label_obj = [x for x in serializer_labels.data if x['id'] == a['label']][0]
+                labels.append('__label__{}'.format(label_obj['text'].replace(' ', '')))
+            data.append('{} {}'.format(' '.join(labels), d['text'].replace('\n', ' ')) if labels else d['text'].replace('\n', ' '))
+        return data
+
+
+class PlainTextRenderer(BaseRenderer):
+    media_type = 'text/plain'
+    format = 'txt'
+
+    def render(self, data, accepted_media_type=None, renderer_context=None):
+        if data is None:
+            return bytes()
+
+        if not isinstance(data, list):
+            data = [data]
+
+        for d in data:
+            yield d + '\n'
```

Comment:
Any reason for using `yield` here? No other renderer i looked at does this.

Besides that, `render()` should return a bytestring, like this:
```python
buffer = BytesIO()
for d in data:
    buffer.write((d + '\n').encode(self.charset))
return buffer.getvalue()
```
`chatset` is defined as `utf-8` in BaseRenderer. Better to set it here explicitly.